### DISCUSSION TITLE
How to run nodejs / npx in home assistant OS installation

### DISCUSSION THREAD

**Discussion by raidboy** [zwave-js/zwave-js#5152] [Q&A]
I am trying to run the zwave controller conversion tool as outlined in [https://github.com/zwave-js/node-zwave-js/tree/master/packages/nvmedit](url), but i can not figure out how to run the npx command. I have home assistant OS on Raspberry PI, but it is not available via SSH CLI.

**Comment by AlCalzone** [comment]
Just wanted to let you know, you'll have more luck asking this on the HA discord or forums right now. I'm on parental leave until January.
